Source/NSubstitute/Core/ICallResultsCache.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NSubstitute.Core
+{
+    public interface ICallResultsCache
```

Comment:
Merged. Actually yes, you were right, wrapper it too tiny so we could live without it :)